bin/hub/logic/maidsPartyNightSinglePlayerAdventure.ts

```diff
@@ -2,21 +2,14 @@
  * Copied from the original bot hub, with permission.
  */
 
-import promClient from "prom-client";
-
 import { wait } from "../utils";
-import * as JMod from "../../jmod"
 
 const SUPERUSERS: number[] = [];
 
 import _ from "lodash";
 import fs from "fs";
 import { LoggingLogic } from "./loggingLogic";
-import { API_Character } from "../../apiCharacter";
-import { API_AppearanceItem, AssetGet, BC_AppearanceItem } from "../../item";
-import { logger } from "../../api";
-import { API_Connector } from "../../apiConnector";
-import { AnyCharacterEvent, BC_Server_ChatRoomMessage } from "../../logicEvent";
+import { API_Character, API_AppearanceItem, AssetGet, BC_AppearanceItem, logger, API_Connector, AnyCharacterEvent, BC_Server_ChatRoomMessage, JMod_allowApplyAppearanceBundle, JMod_importAppearanceBundle, JMod_exportAppearanceBundle, JMod_applyAppearanceBundle } from "bc-bot";
```

Comment:
Of note, I didn't try to keep all the JMod exports under their own JMod object like they were originally.